.github/ISSUE_TEMPLATE.md

```diff
@@ -6,42 +7,43 @@
 - [ ] Functional Validation
 - [ ] UX Validation
 - [ ] Released
-<!--- Please fill out the appropriate template below as best you can. Keep the Progress checklist. -->
 
-------------------------------------------------------------
+<!------------------------------------------------------------------->
+<!-- Please fill out the appropriate template below as best you can. -->
```

Comment:
nit: change template to sections since there is only one template with this change